README.md

```diff
@@ -8,8 +8,13 @@
 This `ember-cli` addon adds a simple flash message service to your app. It's injected into all `Controllers`, `Routes`, `Views` and `Components` by default ([you can change this](#service-defaults)), or lazily injected with `Ember.inject.service`.
 
 ## Installation
-You can install either with `ember install:addon`:
+You can install either with `ember install`:
 
+For Ember CLI >= 0.2.3:
+```shell
```

Comment:
Could you please add a newline before and after the code blocks? It renders the same, but I'd like to keep the formatting consistent